data/human/war reactions.txt

```diff
@@ -0,0 +1,248 @@
+				goto questions
+			label sister
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label sister"
+			`	"Yup," Fiona says, "I blew up her Cruiser! When her ship's escape pods were veering towards a star, we rescued them."`
+			`	Rain continues, "Can you imagine my surprise when I woke up on my sister's new Dreadnought?"`
+			`	"Hey, you got a new Auxiliary commission afterwards," replies Fiona, poking Rain as she says it, "and your whole crew survived. Now we escort each other's ships during investigations!"`
+				goto questions
+			label mercenary
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label mercenary"
+			`	"I meant no offense," says Fiona, "you're quite the hero." She thinks for a second and glances at her sister. "Rain, can you explain?"`
+			`	Rain says, "We're uniformed career officers who must follow orders. Our states own the ships, uniforms, and everything else we use. Our salaries may be meager compared to your earnings, but serving one's state is a rich reward.`
+			`	"You, on the other hand, own your ship and serve whoever you want. To keep up with costs, you have to charge a lot of money for your services. Mercenaries only survive as long as you if they're skilled at both finances and combat."`
+				goto questions
+
+
+mission "War Reactions: Syndicate Orphan"
+	landing
+	minor
+	source
+		government "Syndicate"
+		not attributes "station"
+	to offer
+		has "event: post-war reactions begin"
+		random < 30
+	on offer
+		set "pug media coverage"
+		conversation
+			`	When you exit your ship, you hear sounds of fists falling on skin and a kid yelling for help. Around the corner are three boys hitting a fourth, who just fell on the ground. It's not clear if they're going to stop attacking him.`
+			choice
+				`	"Break it up, kids."`
+				`	(Ignore them and keep walking.)`
+					decline
+			`	The three attackers turn around, looking furious. They eye your gun and run away. "Thanks. I'm Gus," says the fourth boy, as he slowly rises to his feet. He looks about twelve years old.`
+			`	It takes Gus a few seconds to recover. He looks worried and asks, "Can you walk me to the factory? I'm already late for work, and I'm afraid they'll attack me again."`
+			label questions
+			choice
+				`	"Why were those boys attacking you?"`
+					goto attacking
+					to display
+						not "War Reactions: Syndicate Orphan: attacking"
+				`	"Aren't you a little young to work in a factory?"`
+					goto young
+				`	"I'm in a hurry. You're on your own."`
+					decline
+			label attacking
+			action
+				set "War Reactions: Syndicate Orphan: attacking"
+			`	"My sister is a Private, Third Class, in the Syndicate's Ancillary Security Service, which monitors Syndicate-owned workplaces. Those boys are factory workers, and Syndicate security officers aren't nice to factory workers. They can't attack my sister, so they attack me instead."`
```

Comment:
The syndicate doesn't sound like the sort of place that would bind his sister's hands from taking retributive action against these boys, so the justification feels weak.

Reply:
This bit I agree with, though, because even though such weak, ultimately self destructive justifications are indeed often used in real life, to me, this is an instance where fiction must be more sensible in order to not feel strained. I'd suggest coming up with a specific incident of her coming down on their colleagues (as example) for a more direct motivation to lash out at him as a surrogate.

Reply:
The boys can't attack his sister because his sister is part of the Syndicate's security forces. Attacking security officers has rather severe consequences. The mission does not address whether the sister takes revenge against the boys since that isn't relevant to the mission (it would happen afterwards).

Reply:
The justification has been improved. I misunderstood your comments in my prior PR.